spec/client_spec.js

```diff
@@ -419,15 +419,15 @@ describe('Client', function() {
         it('resolves when the request succeeds', function(done) {
           triggerEvent(subject, 'request:' + requestsCount + '.done', response);
           promise.then(function() {
-            expect(doneHandler).to.have.been.calledWith(response.responseArgs[0]);
```

Comment:
Nice, good investigation. In this case perhaps we could convert the node to text before `postMessage` then converting it back to a document node in ZAF SDK. That doesn't solve the problem that we're omitting a bunch of useful information, like request headers (as reported by others). Maybe we could make a minor bump for this :/